http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/HttpRoutesSpec.scala

```diff
@@ -130,6 +139,57 @@ class HttpRoutesSpec
     }
   }
 
+  it should "delete a runtime and disk if deleteDisk is true" in {
+    val runtimeService = new BaseMockRuntimeServiceInterp {
+      override def deleteRuntime(deleteRuntimeRequest: DeleteRuntimeRequest)(
+        implicit as: ApplicativeAsk[IO, AppContext]
+      ): IO[Unit] = IO {
+        val expectedDeleteRuntime =
+          DeleteRuntimeRequest(timedUserInfo, GoogleProject("googleProject1"), RuntimeName("runtime1"), true)
+        deleteRuntimeRequest shouldBe expectedDeleteRuntime
+      }
+    }
+    val routes = fakeRoutes(runtimeService)
+    Delete("/api/google/v1/runtimes/googleProject1/runtime1?deleteDisk=true") ~> routes.route ~> check {
+      status shouldEqual StatusCodes.Accepted
+      validateRawCookie(header("Set-Cookie"))
+    }
+  }
+
+  it should "not delete a runtime and disk if deleteDisk is false" in {
```

Comment:
could we rename this test to "should only delete runtime if deleteDisk is false" or "should not delete disk if deleteDisk is false"?